File: src/easy/DifferenceOfSums.swift
func differenceOfSums(_ n: Int, _ m: Int) -> Int {
  var lhs = 0
  var rhs = 0
  for i in 1...n {
    if i % m == 0 {
      rhs += i
    } else {
      lhs += i
    }
  }
  return lhs - rhs
}

File: lib/easy/difference_of_sums.dart
int differenceOfSums(int n, int m) {
  int lhs = 0;
  int rhs = 0;
  for (int i = 1; i < n + 1; i++) {
    if (i % m == 0) {
      rhs += i;
    } else {
      lhs += i;
    }
  }
  return lhs - rhs;
}
